Formula/c/cidr.rb

```diff
@@ -0,0 +1,22 @@
+class Cidr < Formula
+  desc "A CLI to perform various actions on CIDR ranges"
+  homepage "https://github.com/bschaatsbergen/cidr"
+  url "https://github.com/bschaatsbergen/cidr/archive/refs/tags/v1.0.1.tar.gz"
+  sha256 "445d7197afdaf320c6c39032b85a274b2a31e0984af06ecefa835e5cb2d7a259"
+  license "MIT"
+  head "https://github.com/bschaatsbergen/cidr.git", branch: "main"
+  
+  depends_on "go" => :build
+  
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/bschaatsbergen/cidr/cmd.version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+  
+  test do
+    assert_equal "cidr version #{version}\n", shell_output("#{bin}/cidr --version")
+  end
```

Comment:
We will need a test that exercises some functionality of the app. The version check is good for ensuring the ldflag worked correctly, so that's worth keeping.

```suggestion
  test do
    assert_match version.to_s, shell_output("#{bin}/cidr --version")
  end
```